templates/base.html

```diff
@@ -32,11 +32,11 @@
       </button>
       <div class="navbar-collapse collapse" id="navbarOptions">
         <ul class="navbar-nav ml-auto">
-          <li class="nav-item"><a class="nav-link" href="{% url 'requestgroups:list' %}">Home</a></li>
-          <li class="nav-item"><a class="nav-link" href="{% url 'requestgroups:create' %}">Submit Observation</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Home</a></li>
```

Comment:
This is just a placeholder since this file should go away later right?

Reply:
Yes that's right